packages/backend/src/db/transactionProvider.spec.ts

```diff
@@ -0,0 +1,18 @@
+// import { App, ServiceContextEvents } from '@h4bff/core';
+// import { TransactionProvider } from './transactionProvider';
+// import { Database } from './database';
+
+describe('TransactionProvider', () => {
+  it(`should register 'dispose' callback on context disposal`, () => {
```

Comment:
`it.skip` is a neat trick to skip unfinished / WIP tests and have jest report them